src/compile.ts

```diff
@@ -73,6 +75,7 @@ export default function compile({
               ...tsconfigOverrides,
               target: ecma === 6 ? ts.ScriptTarget.ES2015 : ts.ScriptTarget.ES5,
             },
+            tsconfigSearchPath: path.dirname(entryPoint),
```

Comment:
`tsconfigLocationPath`?

`tsconfigSearchPath` might seem like a path that tsconfig needs/is searching for, for someone not aware of the per component tsconfig.

Reply:
Location makes it sound like it's definitely there, it's search path because it searches in that directory and upwards. Not sure of a better name that conveys that?

Reply:
then maybe just leave it `tsconfigSearchPath`